server/athenian/api/controllers/team_controller.py

```diff
@@ -305,6 +306,63 @@ async def get_team_from_db(
     return team
 
 
+async def teams_recursive_fetch(
+    account_id: int,
```

Comment:
```suggestion
    account: int,
```
(common naming in the project)